src/include/sof/audio/codec_adapter/codec_adapter.h

```diff
@@ -10,7 +10,7 @@
 
 #include <sof/audio/codec_adapter/codec/generic.h>
 
-#define MAX_BLOB_SIZE 4096
+#define MAX_BLOB_SIZE 8192
 /*****************************************************************************/
```

Comment:
- Commit message must specify WHY the change is needed.
- Commit subject should start with ``audio: codec_adapter: ...``